packages/bpk-component-nudger/README.md

```diff
@@ -47,19 +49,104 @@ class App extends Component {
 }
 ```
 
+### Configurable `BpkConfigurableNudger`
+
+```js
+import React, { Component } from 'react';
+import BpkLabel from 'bpk-component-label';
+import { BpkConfigurableNudger } from 'bpk-component-nudger';
+
+class App extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      value: 1,
+    };
+  }
+
+  handleChange = value => {
+    this.setState({ value });
+  };
+
+  compareValues = (a, b): number => {
+    const options = ['economy', 'premium', 'business', 'first'];
+    const [aIndex, bIndex] = [options.indexOf(a), options.indexOf(b)];
+    return aIndex - bIndex;
+  };
+
+  incrementValue = (a): string => {
+    const options = ['economy', 'premium', 'business', 'first'];
```

Comment:
Could you provide some more detail about this suggestion?

Reply:
As in preventing accessing things outside the bound of the array, so like in this example if they didn't set a maximum but kept clicking the + button they'll eventually get `undefined` because they'll go off the end of the array. So we could put an `if` in here to stop that happening.

Reply:
I don't think that scenario would ever happen as max and min are required props so it would never get over/undefined due to there always being the limit value

Reply:
I am having issues seeing how a user could get the function to go out of bounds

Reply:
That's a good point, I didn't realise max and min were required. I'm going to defer to @k0nserv in that case

Reply:
What if there's a bug in our code 😱? It's fairly reasonable behaviour to guard it with an out of bounds check imo